OptimoveSDK/app/src/main/java/com/optimove/android/optimovemobilesdk/MyApplication.java

```diff
@@ -31,11 +34,14 @@ public void onCreate() {
     super.onCreate();
 
     Optimove.initialize(this, new OptimoveConfig.Builder(
-            "WyIxIiwgIjgwYTRhMjI0ZGRkMTRhNDQ4MTNlYzIwZmNkMjAxNjE2IiwgIm1vYmlsZS1jb25maWd1cmF0aW9uLjEuMC4wIl0=",
-            "WzEsInVrLTEiLCI2YjE5OThhYS1lZmM1LTRjODUtYjg4ZC1mMjQzMTE4ODA1NTAiLCJKcTMxVEJ6dmxmVTQxb2xzMXltQVZTSVdjNXlnY3VmbHpjbysiXQ==")
+            "base64_optimove_creds",
```

Comment:
Should maybe remove the base64 prefix? could confuse that there is an additional base64 needed here